packages/ethereum-light-client/src/header.rs

```diff
@@ -0,0 +1,56 @@
+//! This module defines types related to the Header we use for the Ethereum light client
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use ethereum_types::{
+    consensus::{light_client_header::LightClientUpdate, sync_committee::SyncCommittee},
+    execution::account_proof::AccountProof,
+};
+
+/// The header of a light client update
+#[derive(Serialize, Deserialize, JsonSchema, PartialEq, Eq, Clone, Debug, Default)]
+pub struct Header {
+    /// The trusted sync committee
+    pub trusted_sync_committee: TrustedSyncCommittee,
+    /// The consensus update
+    pub consensus_update: LightClientUpdate,
+    /// The account update
+    pub account_update: AccountUpdate,
+}
+
+/// The account update
+#[derive(Serialize, Deserialize, JsonSchema, PartialEq, Eq, Clone, Debug, Default)]
+pub struct AccountUpdate {
+    /// The account proof
+    pub account_proof: AccountProof,
+}
+
+/// The trusted sync committee
+#[derive(Serialize, Deserialize, JsonSchema, PartialEq, Eq, Clone, Debug, Default)]
+#[allow(clippy::module_name_repetitions)]
+pub struct TrustedSyncCommittee {
+    /// The trusted slot
+    pub trusted_slot: u64,
+    // TODO:
+    /// Active sync committee
+    // pub active_sync_committee: ActiveSyncCommittee,
+    /// The current sync committee
+    pub sync_committee: ActiveSyncCommittee,
+}
+
+/// The active sync committee
+#[derive(Serialize, Deserialize, JsonSchema, PartialEq, Eq, Clone, Debug)]
+#[allow(clippy::module_name_repetitions)]
+pub enum ActiveSyncCommittee {
+    /// The current sync committee
+    Current(SyncCommittee),
+    /// The next sync committee
+    Next(SyncCommittee),
+}
+
+impl Default for ActiveSyncCommittee {
```

Comment:
I just removed Default from here and the consensus state. It was just used to make the tests a little simpler. I've kept it for ClientState just because it doesn't require anything extra and make some test shorter ^^